py/range.go

```diff
@@ -104,6 +108,42 @@ func (r *Range) M__iter__() (Object, error) {
 	}, nil
 }
 
+func (r *Range) M__str__() (Object, error) {
+	return r.M__repr__()
+}
+
+func (r *Range) repr() (Object, error) {
+	var b strings.Builder
```

Comment:
Since strings.Builder is not supported at go < 1.10
You need to use version constraints.
Read this article. https://medium.com/@theckman/version-constraints-and-go-c9309be15773

example: 
https://github.com/corona10/goimagehash/blob/master/imagehash18.go
https://github.com/corona10/goimagehash/blob/master/imagehash19.go